docs/reference/remap/get_env_var.cue

```diff
@@ -0,0 +1,31 @@
+package metadata
+
+remap: functions: get_env_var: {
+	arguments: [
+		{
+			name:        "name"
+			description: "Name of the environment variable."
+			required:    true
+			type: ["string"]
+		},
+	]
+	return: ["string"]
+	category: "System"
+	description: #"""
+		Get an environment variable. If not exists, a `Call` error will be raised.
+		"""#
+	examples: [
+		{
+			title: "Get environment variable"
+			input: log: {}
+			source: #"""
+				.home = get_env_var("HOME")
```

Comment:
@JeanMertz is this right? Should this instead have an `!`? That is: `.home = get_env_var!("HOME")`

Reply:
It should

Reply:
@jszwedko  Hi, what does `!` mean? Code seems to run fine without it. Is it needed for `.not_found = get_env_var("SOME_VAR") ?? "default"` too?

Reply:
The `!` can be used to have the script fail if the the function returns an error (in this case, the env var doesn't exist). I believe that it should be required if you are simply doing `.home = get_env_var("HOME")` as otherwise remap should complain that the error wasn't handled.

You are right, though, it works without:

```
$ echo '{}' | ./target/debug/vrl '.home = get_env_var!("HOME2")'
remap error: remap error: function call error: environment variable not found
```

@JeanMertz am I misunderstanding? Or is there still some uncompleted work?

Reply:
I tested `parse_json`, which is another fallible function, same result as here.

Reply:
Yeah, I think there might still be some pending work on remap here but I'll let @JeanMertz confirm.

Reply:
Correct. That work isn’t merged yet, but will be soon. I thought your comment was about future-proofing this example.

Reply:
Ok, I'll keep the `!`.